modules/express/src/typedRoutes/api/v2/ofcSignPayload.ts

```diff
@@ -0,0 +1,31 @@
+import * as t from 'io-ts';
+import { Json, NonEmptyString } from 'io-ts-types';
+import { httpRoute, httpRequest, optional } from '@api-ts/io-ts-http';
+
+/**
+ * Sign an arbitrary payload using an OFC trading account key.
+ * @operationId express.ofc.signPayload
+ * POST /api/v2/ofc/signPayload
+ */
+export const OfcSignPayloadBody = {
+  walletId: NonEmptyString,
```

Comment:
Missing jsdocs. In this case, since there is no platform.yaml to reference, lets use the jsdocs from the underlying code.